pySDC/core/sweeper.py

```diff
@@ -263,3 +263,15 @@ def level(self, L):
     @property
     def rank(self):
         return 0
+
+    def updateVariableCoeffs(self, k):
+        """
+        Eventually update QDelta implicit coefficients if variable ...
```

Comment:
Why `Eventually`? That sounds like it will come in the future

Reply:
Nah, it's just that most preconditioners keep the same coefficient for different `k`

Reply:
Ah, I see! This is a false friend, I guess. https://www.oed.com/search/dictionary/?scope=Entries&q=eventually

Reply:
`Potentially`?

Reply:
Well, this one was vicious ...